Zend/tests/closure_041.phpt

```diff
@@ -53,9 +53,9 @@ $d = $nonstaticScoped->bindTo(null); $d(); echo "\n";
 $d->bindTo($d);
 
 echo "After binding, with same-class instance for the bound ones", "\n";
-$d = $staticUnscoped->bindTo(new A); $d(); echo "\n";
+$d = $staticUnscoped->bindTo(new A); /* $d(); */ echo "\n";
```

Comment:
is it necessary to change existing tests? does this reflect some change in behavior?